frontend/src/components/common/button/button.tsx

```diff
@@ -1,17 +1,38 @@
+import { AppRoute } from 'common/enums/enums';
 import { FC } from 'common/types/types';
+import { getValidClasses } from 'helpers/helpers';
+import { useNavigate } from 'hooks/hooks';
 
 import styles from './styles.module.scss';
 
 type Props = {
   label: string;
+  btnColor?: 'blue' | 'gray';
```

Comment:
```suggestion
  color?: 'blue' | 'gray';
```

Reply:
@Liza-Veis @what1s1ove guys, let's make a final decision on how it should be called because the previous suggestion was to change it to btnColor. thanks

Reply:
btnColor good!